config/hirte-default.conf

```
@@ -0,0 +1,50 @@
+# This is the default configuration values for hirte and hirte-agent
+
+#
+# Common options - following options are usable for both hirte and hirte-agent
+#
+
+# The level used for logging. Supported values are: DEBUG, INFO, WARN and ERROR.
+LOG_LEVEL=INFO
+
+#
+# The target where logs are written to. Supported values are: journald and
+# stderr.
+LOG_TARGET=journald
+
+#
+# If this flag is set to true, no logs are written by hirte.
+LOG_IS_QUIET=false
+
+#
+# The port the manager listens on to establish connections with the hirte
+# agents.
+MANAGER_PORT=2020
+
+
+#
+# hirte options - following options are usable only for hirte manager.
+#
+
+#
+# A comma separated list of unique hirte-agent names. These are used as
+# a whitelist to validate any register request from each hirte-agent.
+VALID_NODE_NAMES=
```

```suggestion
EXPECTED_NODES=
```
For the whole system to be running, `hirte` requires those nodes to be present (which is currently not enforced) and expects them to be there. Valid would mean its ok if some are missing.

---

Yeah, but I'm not sure `EXPECTED` tells the whole story, so how about `ALLOWED_NODE_NAMES` or `APPROVED_NODE_NAMES`?

---

True. `ALLOWED_NODE_NAMES` sounds good :)

---

Changed